councilmatic_core/migrations/0048_post_shape.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 2.1.9 on 2019-06-26 17:10
+
+import django.contrib.postgres.fields.jsonb
+import django.core.files.storage
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0004_auto_20171005_2028'),
+        ('councilmatic_core', '0047_update_filepath'),
+    ]
+
+    operations = [
+        migrations.DeleteModel(
+            name='Post',
+        ),
```

Comment:
For some reason, this migration got generated as a `DeleteModel` operation followed by a `CreateModel` operation. Will this cause a problem for instances migrating existing data?

Reply:
should be fine.